ports/nrf/boards/actinius_icarus/mpconfigboard.mk

```diff
@@ -2,5 +2,6 @@ MCU_SERIES = m33
 MCU_VARIANT = nrf91
 MCU_SUB_VARIANT = nrf9160
 LD_FILES += boards/nrf9160_1M_256k.ld
+MICROPY_FATFS = 1
```

Comment:
Should it be `MICROPY_VFS_FAT`, to allow `extmod/extmod.mk` to include the sources?

Reply:
Ah, that looks like the all-ports version of the same thing. `MICROPY_FATFS` includes the same files directly in micropython/ports/nrf/Makefile

Reply:
But yes, defining `MICROPY_FATFS = 1` in the mk file means it doesn't need to be defined in the header as well so does double duty and makes it more compatible with other ports.